http/typescript/fetch/package.json

```diff
@@ -34,7 +34,7 @@
     "web-streams-polyfill": "^3.2.0"
```

Comment:
I believe with this change we don't need the web streams polyfill anymore, correct? if so, can you remove it from the packages please? also some of the package updates you've made might conflict with dependabot PRs, might be worth rebasing

Reply:
Removed the polyfill